.travis.yml

```diff
@@ -1,14 +1,8 @@
 language: node_js
 node_js:
-   - "6"
-   - "7"
-   - "8"
-   - "9"
-env:
-  - CXX=g++-4.8
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - g++-4.8
+  - "6"
+  - "8"
+  - "10"
+script:
+  - yarn test
```

Comment:
is it possible to use npm instead?

Reply:
yes; travis is auto-detecting the yarn.lock file for install so i kept the streak going. i've updated config to use npm.